<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-T9TC6QDG" failures="1" tests="7" name="BillHub App Test Cases" time="66.825" errors="0" timestamp="2024-02-26T15:09:43 IST">
  <testcase classname="com.billhub.qa.testcases.ReasonPageTest" name="addNewReasonWithValidDataTest" time="1.234"/>
  <testcase classname="com.billhub.qa.testcases.ReasonPageTest" name="addNewReasonWithInvalidDataTest" time="13.802"/>
  <testcase classname="com.billhub.qa.testcases.ReasonPageTest" name="addNewReasonWithoutDataTest" time="10.419"/>
  <testcase classname="com.billhub.qa.testcases.ReasonPageTest" name="addNewReasonWithDuplicateDataTest" time="1.263">
    <failure type="java.lang.AssertionError" message="Duplicate data was added. expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: Duplicate data was added. expected [false] but found [true]
at com.billhub.qa.testcases.ReasonPageTest.addNewReasonWithDuplicateDataTest(ReasonPageTest.java:77)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- addNewReasonWithDuplicateDataTest -->
  <testcase classname="com.billhub.qa.testcases.ReasonPageTest" name="updateReasonTest" time="5.18"/>
  <testcase classname="com.billhub.qa.testcases.ReasonPageTest" name="searchByReasonNameTest" time="1.332"/>
  <testcase classname="com.billhub.qa.testcases.ReasonPageTest" name="searchByReasonCodeTest" time="0.27"/>
</testsuite> <!-- BillHub App Test Cases -->
